src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpTraceExporterHelperExtensions.cs

```diff
@@ -72,6 +75,7 @@ public static TracerProviderBuilder AddOtlpExporter(this TracerProviderBuilder b
 
             OtlpExporterOptions.RegisterOtlpExporterOptionsFactory(services);
             services.RegisterOptionsFactory(configuration => new SdkLimitOptions(configuration));
+            services.TryAddSingleton<OtlpExporterTransmissionHandler<OtlpCollector.ExportTraceServiceRequest>>();
```

Comment:
Do not intend to include this as part of this PR. Wiring up exporters will be part of separate PRs.